src/lib/y2storage/clients/inst_disk_proposal.rb

```diff
@@ -97,7 +97,7 @@ def guided_setup(settings)
       end
 
       def expert_partitioner
-        ExpertPartitioner::MainDialog.new.run
+        Y2Partitioner::Clients::Main.run
```

Comment:
Would it be too picky to say that, since we don't have yast2-partitioner as a dependency, we should detect here if Y2Partitioner is not available and show an error instead of trying to run it?